BWEM/src/graph.cpp

```diff
@@ -176,7 +176,7 @@ void Graph::CreateChokePoints()
 			for (const Area * pA : BlockedAreas)
 			for (const Area * pB : BlockedAreas)
 			{
-				if (pB == pA) break;	// breaks symmetry
+				if (pB == pA || !pA || !pB) break;	// breaks symmetry
```

Comment:
Looks like somehow `null` in the `BlockedAreas`. Is this correct ? Probably proper fix would be to find out why it is inserted in the first place. Does this happening just when loading map, or under some specific scenario

Reply:
Let me take a look, looks like perfect example for test 😄